pygmt/src/x2sys_cross.py

```diff
@@ -235,6 +235,7 @@ def x2sys_cross(tracks=None, outfile=None, **kwargs):
                     header=2,  # Column names are on 2nd row
                     comment=">",  # Skip the 3rd row with a ">"
                     parse_dates=[2, 3],  # Datetimes on 3rd and 4th column
+                    date_format="ISO8601",
```

Comment:
This `date_format` parameter only works with `pandas>=2.0`. Do we want to set a minimum version of pandas, or make things backwards compatible with pandas=1.x?

Reply:
> make things backwards compatible with pandas=1.x?

This would be better.

Reply:
Ok, compatibility with pandas=1.x and 2.x should be done in fa2a6d3437188948ae6232a8c6b2140986799f36 and 146a7f46cfd7abec9450019606d844fbd2ea8a76. I've also changed ci_tests_legacy to run with `pandas<2` as a check.